### PR TITLE
Pysnptools upgrade and associated changes

### PR DESCRIPTION
**Pull request by eric-czech**
Pysnptools had an update a few days ago and the most significant improvement it seems to introduce is reading PLINK calls as 8-bit ints rather than 32-bit floats.  This PR includes breaking changes (from pysnptools 0.4.19) to use these new parameters and also adds support for PLINK files with different names.

### CODE REVIEW

File: sgkit_plink/pysnptools.py [hunk 1]
```diff
@@ -137,18 +136,20 @@ def read_plink(
...
-    path : PathType
-        Path to PLINK file set.
-        This should not include a suffix, i.e. if the files are
-        at `data.{bed,fam,bim}` then only 'data' should be
-        provided (suffixes are added internally).
+    path : Union[PathType, Tuple[PathType, PathType, PathType]]
+        Path to PLINK file set.  This can either be one path or 3.
+        If one path, then that path should not include a suffix such
+        that if the files are at `data.{bed,bim,fam}` then only 'data'
+        should be provided (suffixes are added internally). If 3 paths,
```
Comment (ravwojdyla) [inline sgkit_plink/pysnptools.py]: Depending on an order in a tuple seems rather error prone, wouldn't it be more idiomatic to have 4 flags:

```
bed_path: Optional[PathType],
bim_path: Optional[PathType],
fam_path: Optional[PathType],
path_prefix: Optional[PathType],  # {bed,bim,fam} are added

# check either all([bed_path, bim_path, fam_path]) or path_prefix
```

and with that order you can still do a convenient (but unsafe) shortcut:

```python
read_plink(*paths, sep=blah)
```

?

**Review by ravwojdyla — APPROVED** [review]
Approving since I don't have a strong opinion on the single comment, I leave that up to you.

File: sgkit_plink/pysnptools.py (hunk 1 above)
Comment (tomwhite) [reply sgkit_plink/pysnptools.py]: It looks like this change is independent of the pysnptools upgrade? If so, then it's the kind of change that could go in a separate PR.

Comment (jeromekelleher) [reply sgkit_plink/pysnptools.py]: I agree with @tomwhite - this is a significant UI decision which is independent of the pysnptools upgrade.

Comment (eric-czech) [reply sgkit_plink/pysnptools.py]: I see it as trivial this early but happy to pull it out: https://github.com/pystatgen/sgkit-plink/pull/15

**Review by ravwojdyla — APPROVED** [review]

**Pull request merged**